app/views/hyrax/dashboard/sidebar/_repository_content.html.erb

```diff
@@ -15,8 +15,6 @@
     <%= menu.nav_link(bulkrax.importers_path) do %>
       <span class="fa fa-cloud-upload" aria-hidden="true"></span> <span class="sidebar-action-text"><%= t('bulkrax.admin.sidebar.importers') %></span>
     <% end %>
-    <%= menu.nav_link(bulkrax.exporters_path) do %>
-      <span class="fa fa-cloud-download" aria-hidden="true"></span> <span class="sidebar-action-text"><%= t('bulkrax.admin.sidebar.exporters') %></span>
-    <% end %>
+    <!-- REMOVED LINK TO EXPORT. Find the code here to reimplement: https://github.com/samvera/hyku/blob/97e392fd26196738f6bb846cf91167f961e48d4e/app/views/hyrax/dashboard/sidebar/_repository_content.html.erb#L19-->
```

Comment:
You dont need these kind of comments because the code is all with the Git history and anyone can come back and see the history.

Reply:
Yeah well I am in a generous mood and am making it easy for people